src/main/java/bdv/fx/viewer/render/RenderingModeController.java

```diff
@@ -0,0 +1,76 @@
+package bdv.fx.viewer.render;
+
+import java.lang.invoke.MethodHandles;
+import java.util.Timer;
+import java.util.TimerTask;
+
+import org.apache.commons.lang.NotImplementedException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import net.imglib2.realtransform.AffineTransform3D;
+import net.imglib2.ui.TransformListener;
+
+public class RenderingModeController implements TransformListener<AffineTransform3D> {
+
+	private static Logger LOG = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+
+	public static enum RenderingMode {
+		MULTI_TILE,
+		SINGLE_TILE
+	}
+
+	private static final int[] DEFAULT_TILE_SIZE = {250, 250};
+
+	private final RenderUnit renderUnit;
+	private RenderingMode mode;
+
+	private final int modeSwitchDelay = 250;
+	private final Timer modeSwitchTimer;
+	private TimerTask modeSwitchTimerTask;
```

Comment:
I ended up not using this `Timer` here at all. A different logic was implemented in https://github.com/saalfeldlab/paintera/pull/200/commits/7ced12a6cd9c9bf81ae0f29d8e9de463336145eb

Sorry for cluttering the changeset with intermediate changes that were removed in later commits. Maybe it's easier to review this PR in full, or group the first few commits as a single changeset: https://github.com/saalfeldlab/paintera/pull/200/files/7ced12a6cd9c9bf81ae0f29d8e9de463336145eb